tests/unit/data-access/FetchSearchEntityRepository.spec.ts

```diff
@@ -89,16 +159,52 @@ describe( 'FetchSearchEntityRepository', () => {
 		expect( repo.searchProperties( 'instance' ) ).rejects.toThrow( expectedError );
 	} );
 
-	it( 'throws an error if there is a network problem', () => {
+	it( 'throws an error if there is a server side problem fetching an item value', () => {
 		const repo = new FetchSearchEntityRepository(
 			'eo',
 			'https://example.com/w/api.php',
 		);
-		window.fetch = jest.fn().mockImplementation( () => Promise.reject() );
+		window.fetch = jest.fn().mockImplementation( () => Promise.resolve( {
+			ok: false,
+			status: 500,
+			statusText: 'Server Error',
+		} ) );
+
+		const expectedError = new TechnicalProblem( '500: Server Error' );
 
-		const expectedError = new TechnicalProblem( 'Network error' );
+		expect( repo.searchItemValues( 'John Smith' ) ).rejects.toThrow( expectedError );
+	} );
+
+	it( 'throws an error if there is a server side problem fetching a property', () => {
```

Comment:
No, this test should be about network errors, not server errors

Reply:
I deleted that one by accident it seems.